src/inputs.ts

```diff
@@ -11,6 +11,14 @@ const getBool = (name: string): boolean => {
   return actionsCore.getBooleanInput(name);
 };
 
+/**
+ * Convert a comma-separated string input into an array of strings, stripping all whitespace from the string.
```

Comment:
IMHO the whitespace stripping should be up to the consumer.

Reply:
Okay, I'll have separate functions

Reply:
Actually, I'll make it an optional function arg